Week1/homework/create-database.js

```diff
@@ -0,0 +1,53 @@
+const util = require('util');
+const mysql = require('mysql');
+const countries = require('./countries.js');
+const cities = require('./cities.js');
+
+const connection = mysql.createConnection({
+  multipleStatements: true,
+  host: 'localhost',
+  user: 'hyfuser',
+  password: '7566',
+  database: 'userdb'
+});
+
+const execQuery = util.promisify(connection.query.bind(connection));
+
+async function seedDatabase() {
+  const CREATE_DATABASE_AND_LOGIN = `
+    CREATE DATABASE IF NOT EXISTS world; USE world`;
```

Comment:
Or, you can take out the table creation code, put them in another place where you tell `mysql.js` to use `database: world`.

Reply:
But I especially used this syntax to put them together.
Is there anything wrong with it ?